spec/active_triples/relation_spec.rb

```diff
@@ -2,6 +2,27 @@
 require 'rdf/isomorphic'
 
 describe ActiveTriples::Relation do
+  let(:parent_resource) { double("parent resource", reflections: {}) }
+
+  subject { described_class.new(parent_resource, double("value args") ) }
+
+  describe "#predicate" do
```

Comment:
Agreed. I'll make it public. That seems like useful info to have on your `Relation`.